Cargo.toml

```diff
@@ -16,6 +16,9 @@ license = "MIT"
 
 description = "A contiguous region of bytes, useful for I/O operations."
 
+[dependencies]
+num = "*"
```

Comment:
It's needed for `num::traits::PrimInt` (specifically the `from_be` and `from_le` methods), since `std::num::Int` is deprecated.